api/core/indexing_runner.py

```diff
@@ -282,9 +282,12 @@ def indexing_estimate(
         if doc_form and doc_form == "qa_model":
             if len(preview_texts) > 0:
                 # qa model document
-                response = LLMGenerator.generate_qa_document(
-                    current_user.current_tenant_id, preview_texts[0], doc_language
-                )
+                if "Q00001:" in preview_texts[0] and "A00001:" in preview_texts[0]:
```

Comment:
This one doesn't seem very generic.

Reply:
Is this the separator? "Q00001:" and "A00001:"?   I can change to a common separator.

Reply:
It is only a runtime delimiter variable and does not actually store

Reply:
@JohnJyong